gamd/parser.py

```diff
@@ -185,11 +192,26 @@ def parse_charmm_tag(input_files_tag):
             charmm_config.topology = assign_tag(charmm_tag, str)
         elif charmm_tag.tag == "coordinates":
             charmm_config.coordinates = assign_tag(charmm_tag, str)
+            if "type" in charmm_tag.attrib:
+                type_attrib = charmm_tag.attrib["type"]
+                charmm_config.coordinates_filetype = type_attrib
+
+        elif charmm_tag.tag == "box_vectors":
+            vecA,vecB,vecC = charmm_tag.text.split()
+            charmm_config.box_vectors = [float(vecA), float(vecB), float(vecC)]
+
         elif charmm_tag.tag == "parameters":
-            charmm_config.parameters = []
-            for xml_params_filename in charmm_tag:
-                charmm_config.parameters.append(
-                    assign_tag(xml_params_filename, str))
+            # parsing list of parameter files like CHARMM-GUI does
```

Comment:
@lvotapka note that the parameter parsing used here loops over all parameter files listed in a "toppar.str" file as defined by the user. Multiple parameter files are accepted.

I believe this approach is preferable because: 1) it is commonly used in CHARMM and is compatible with CHARMM-GUI inputs for OpenMM; and 2) CHARMM commonly used +10 parameter files and listing all of them in the XML input seems unnecessary to me.